api/v1beta1/foundationdbcluster_types_test.go

```diff
@@ -2411,11 +2413,11 @@ var _ = Describe("[api] FoundationDBCluster", func() {
 			Entry("With a placeholder",
 				testCase{
 					processAddr: ProcessAddress{
-						StringAddress: "$POD_IP",
+						StringAddress: fmt.Sprintf("$%s", fdbv1beta2.EnvNamePodIP),
```

Comment:
```suggestion
						StringAddress: string(fdbv1beta2.EnvNamePodIP),
```

Is there a reason to use `fmt.Sprintf`?

Reply:
the dollar sign

Reply:
Seems like my brain ignored the `$` :)

Reply:
oh mine did too until the tests all failed 🤣